src/component/AddBook.tsx

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable no-console */
+/* eslint-disable import/no-default-export */
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
```
/* eslint-disable no-console */
/* eslint-disable import/no-default-export */
/* eslint-disable @typescript-eslint/no-unused-vars */
```
Why don't we disable eslint all together?!!

Here are some stuffs:
 - suger without sweet
 - camera without chip
 - laptop without adapter

If we are using some tools, then we need to do it properly. Otherwise we need to skip it. For this instance, we can't skip it.